web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
```

Comment:
>It could be a lot simpler to just hardcode these?

What would you export otherwise?

There's no point exporting a controller function to be used in `myModule.controller('SomeCtrl', importedCtrlFunction)`, because that's a separation of code that is more confusing than helpful. So instead, each file registers its own `angular.module` and angular components (like we've done elsewhere), and then exports the module and component names so they can be depended on by other files.

If they're being exported (the only import/export setup that makes sense to me), we may as well use them.